types/papaparse/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for PapaParse v4.5
+// Type definitions for PapaParse v4.5.1
```

Comment:
I'm not convinced this is needed

Reply:
Will it overwrite the existing package version if that’s removed?

It changes nothing functionally so can definitely be removed I was just unsure of DT versioning policy.

Reply:
This references the version of paparse those typing are for, not the version of the typings themselves.

Reply:
Easy then i'll remove it :)